pkg/server/nfs/nfs_server.go

```diff
@@ -163,7 +163,9 @@ func getUpdatedGaneshConfig(config []byte) []byte {
 		LogPath: logPath,
 	}
 
-	template.Must(template.New("Ganesha_Config").Parse(string(config))).Execute(&tmplBuf, tmplVals)
+	if err := template.Must(template.New("Ganesha_Config").Parse(string(config))).Execute(&tmplBuf, tmplVals); err != nil {
+		logrus.WithError(err).Fatal("Failed to parse ganesha config")
```

Comment:
Let's return the error to let the caller handle instead of failing the program here.

Reply:
Since this function doesn't return error orginally, I update to log a warning message here. We can see whether to improve it if this error happens frequently.

Reply:
cc @james-munson @derekbit